src/zepben/ewb/services/network/tracing/networktrace/actions/equipment_tree_builder.py

```diff
@@ -43,6 +44,26 @@ def __init__(self):
     def roots(self) -> Generator[TreeNode[ConductingEquipment], None, None]:
         return (r for r in self._roots.values())
 
+    def recurse_nodes(self) -> Generator[TreeNode[ConductingEquipment], None, None]:
```

Comment:
this is not used anywhere at the moment - It was part of how i was lazy-building leaves before, but ive had a few times ive needed to write a similar method/function lately.

Happy to remove it, left it in as an option.